h/buildext.py

```diff
@@ -188,22 +198,22 @@ def build_chrome(args):
 
         chrome-extension://<extensionid>/public
     """
-    paster.setup_logging(args.config_uri)
-
-    os.environ['WEBASSETS_BASE_DIR'] = os.path.abspath('./build/chrome/public')
-    if args.assets is not None:
-        os.environ['WEBASSETS_BASE_URL'] = args.assets
+    base_url = args.service_url
```

Comment:
This is now mightily confusing, because we have two names for the same thing and it's not always clear which one is supposed to have a trailing slash and which one isn't.

If this is being passed around internally as `base_url`, I'm not sure we should have renamed the external argument to ``--service``.

If the only thing we really need is the service URL, then we probably shouldn't be chopping off the trailing slash here and then adding it back later with `url_with_path`.